composer.json

```diff
@@ -39,7 +39,6 @@
         "aws/aws-sdk-php": "^2.4",
         "doctrine/cache": "^1.11|^2.0",
```

Comment:
do we still need this or can we get rid of it, now that CacheResolver has been removed?

Reply:
you're right, there are no references to doctrine cache, thanks!